index.html

```diff
@@ -1,7 +1,115 @@
+  <head>
+    <title>Why Choose Computer Science?</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  <div id="container">
+
+    <div class="topbar">
+      <div class="topbar-nav">
+        <a href="index.html">Home Page</a>
+        <a href="Whystudy.html">Why?</a>
+        <a class="dropdown-btn" href="what-its-like.html">What is it like?</a>
+        <div class="dropdown-container">
+          <a href="cs-majors.html">Possible Majors</a>
+          <a href="gamingAndAnim.html">Gaming</a>
+          <a href="Cybersec.html">Cybersecurity</a>
+          <a href="business.html">Business</a>
+        </div>
+        <a href="careers.html">Careers</a>
+        <a href="Howtoprepare.html">How do I prepare?</a>
+        <a href="how-we-made-this.html">How we made this</a>
+        <a href="learnMore.html">Learn More</a>
+      </div>
+    </div>
+    <script src="dropdown-menu.js"></script>
+    <h1 class = "title">Studying Computer Science<br><br></h1>
+    <p></p>
+    <br>
+    <div class = "section">
+      <br>
+      <a href="index.html" class = "subtitle">The Art of Programming</a>
+      <p class ="generalDark">
+        &emsp;Computer science is the art of communication through logic. It is <br>
+        &emsp;about understanding how to make a logical machine do what you desire,<br>
+        &emsp;and often times involves enabling the machine to communicate with an <br>
+        &emsp;end user. This is accomplished through the use of various programming <br>
+        &emsp;languages, and using a corresponding compiler to translate the written<br>
+        &emsp;, human readable "code" into electrical pulses. These pulses are read<br>
+        &emsp;by humans as a sequence of ones and zeros, referred to as binary. <br>
+        &emsp;These electrical signals can be transferred, recorded, interpreted, <br>
+        &emsp;edited, and operated on by cicrutry and mechanisms in the computer. <br>
+        &emsp;This includes handling data (numbers and letters) or reading and <br>
+        &emsp;ennacting instructions and operations from a computer's instruction set.<br>
+        &emsp;These instructions are the opposite binary counterpart of the code, <br>
+        &emsp;and sequences of them make up the logic that drives the computer and its<br> 
+        &emsp;applications. These instructions thereby determine how end users <br>
+        &emsp;interact with there machines.<br>
+      </p>
+    </div>
+
+    <br><br><br><br><br>
+    <div class = "top">
+        <a href="Whystudy.html" class = "subtitle">Why Study Computer Science?</a>
+        <p class="generalLight">
+          &emsp;Computer science is a rising industry that constantly requires a new<br> 
+          &emsp;influx of talent. Between its wealth of career paths (including <br>
+          &emsp;application development, cybersecurity, and game development), its <br>
+          &emsp;pay, and its intrinsic interest. Computer science may well be the<br>
```

Comment:
Fix incomplete sentence
```suggestion
          &emsp;pay, and its intrinsic interest, computer science may well be the<br>
```